src/throttle/simple_throttle.py

```diff
@@ -0,0 +1,26 @@
+class SimpleRateThrottle(BaseThrottle):
+    cache = default_cache
+    timer = time.time
+    cache_format = 'throttle_%(scope)s_%(ident)s'
+    scope = None
+    THROTTLE_RATES = api_settings.DEFAULT_THROTTLE_RATES
```

Comment:
where the instance of this variable `api_settings`?

Reply:
Fixed with 5f03f11

Reply:
Alright, good to go